themes/default/content/_index.md

```diff
@@ -40,39 +40,193 @@ overview:
+        });
+
+        // Export some values for use elsewhere
+        export const kubeconfig = eksCluster.kubeconfig;
+        export const vpcId = eksVpc.vpcId;
+    - title: __main__.py
+      language: python
+      code: |
+        import pulumi
+        import pulumi_awsx as awsx
+        import pulumi_eks as eks
+
+        # Create a VPC for the EKS cluster
+        eks_vpc = awsx.ec2.Vpc("eks-vpc",
+            enable_dns_hostnames=True,
+            cidr_block="10.0.0.0/16")
+
+        # Create the EKS cluster
+        eks_cluster = eks.Cluster("eks-cluster",
+            vpc_id=eks_vpc.vpc_id,
+            public_subnet_ids=eks_vpc.public_subnet_ids,
+            private_subnet_ids=eks_vpc.private_subnet_ids,
+            instance_type="t2.medium",
+            desired_capacity=3,
+            min_size=3,
+            max_size=6,
+            node_associate_public_ip_address=False)
+
+        # Export values to use elsewhere
+        pulumi.export("kubeconfig", eks_cluster.kubeconfig)
+        pulumi.export("vpcId", eks_vpc.vpc_id)
+    - title: main.go
+      language: go
+      code: |
+            package main
+
+            import (
+              "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ec2"
+              "github.com/pulumi/pulumi-eks/sdk/go/eks"
+              "github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+              "github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+            )
+
+            func main() {
+              pulumi.Run(func(ctx *pulumi.Context) error {
+                // Create a new VPC, subnets, and associated infrastructure
+                vpcNetworkCidr := "10.0.0.0/16"
+                eksVpc, err := ec2.NewVpc(ctx, "eks-vpc", &ec2.VpcArgs{
+                  EnableDnsHostnames: pulumi.Bool(true),
+                  CidrBlock:          &vpcNetworkCidr,
+                })
+                if err != nil {
+                  return err
+                }
+
+                // Create a new EKS cluster
+                eksCluster, err := eks.NewCluster(ctx, "eks-cluster", &eks.ClusterArgs{
+                  VpcId:                        eksVpc.VpcId,
+                  PublicSubnetIds:              eksVpc.PublicSubnetIds,
+                  PrivateSubnetIds:             eksVpc.PrivateSubnetIds,
+                  InstanceType:                 pulumi.String("t2.medium"),
+                  DesiredCapacity:              pulumi.Int(3),
+                  MinSize:                      pulumi.Int(3),
+                  MaxSize:                      pulumi.Int(6),
+                  NodeAssociatePublicIpAddress: pulumi.Bool(false),
+                })
+                if err != nil {
+                  return err
+                }
+
+                // Export some values in case they are needed elsewhere
+                ctx.Export("kubeconfig", eksCluster.Kubeconfig)
+                ctx.Export("vpcId", eksVpc.VpcId)
+                return nil
+              })
+            }
+    - title: MyStack.cs
```

Comment:
C# seems to have a different width setting for that module - so it changes the width of everything when it is clicked on